roles/openstack-stack/templates/heat_stack.yaml.j2

```diff
@@ -540,7 +553,11 @@ resources:
     properties:
       count: {{ num_masters }}
       resource_def:
+{% if use_bastion|bool %}
+        type: server_nofloating.yaml
```

Comment:
it works for me with a single master w/o a floating IP. How could I check the case for which it fails?